webapp/components/form/InputChips/InputChips.js

```diff
@@ -0,0 +1,100 @@
+import './InputChips.scss'
+
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Dropdown from '../Dropdown'
+
+import { useInputChips, State } from './store'
+import Chip from './Chip'
+
+const InputChips = (props) => {
+  const {
+    items,
+    itemKey,
+    itemLabel,
+    selection,
+    requiredItems,
+    autocompleteMinChars,
+    readOnly,
+    disabled,
+    validation,
+    onChange,
+    onItemAdd,
+    onItemRemove,
+  } = props
+
+  const { state, Actions } = useInputChips({
+    itemKey,
+    itemLabel,
+    onChange,
+    onItemAdd,
+    onItemRemove,
+  })
+
+  const showDropdown = !readOnly
+
+  return (
+    <div className="form-input-chip">
+      {selection.map((item) => (
+        <Chip
+          key={State.getItemKey(state)(item)}
+          item={item}
+          itemKey={State.getItemKey}
+          itemLabel={State.getItemLabel(state)(item)}
+          onDelete={Actions.removeItem({ selection, state })}
+          canBeRemoved={selection.length > requiredItems}
+          readOnly={readOnly}
+        />
+      ))}
+
+      {showDropdown && (
+        <Dropdown
+          items={items}
```

Comment:
the selected items should be excluded from the dropdown items...

Reply:
where do you think that is better put this filter, pass it as prop to the Dropdown?